morpheus-graphql-client/src/Data/Morpheus/Client/Internal/TH.hs

```diff
@@ -118,6 +118,7 @@ mkEntryWith conName f FieldDefinition {fieldName} =
 -- WAS WAS (User name id)
 -- >>>
 destructRecord :: TypeName -> [FieldDefinition cat s] -> PatQ
+destructRecord _ [] = wildP
```

Comment:
why we need that?

Reply:
I changed this to fix the wrong `ToJSON` instance generation, but looks like this is not the place for the fix. It is the order of pattern matches in `deriveToJSON` that is the culprit, I just pushed a commit to fix the order in there and reverted this one.